src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write code here
+const inputs = document.querySelectorAll('input');
+const inputClass = 'field-label';
+const firstLatterUp = (word = '') => {
+  if (word.length > 0) {
+    return (
+      word.charAt(0).toUpperCase() + word.slice(1)
+    );
+  } else {
+    return word;
+  }
+};
+
+[...inputs].forEach(input => {
```

Comment:
```suggestion
inputs.forEach(input => {
```
forEach is a non-mutating method, no need for copy